package.json

```diff
@@ -25,13 +25,14 @@
     "lib": "lib"
   },
   "dependencies": {
-    "appium-base-driver": "^2.3.0",
-    "appium-support": "^2.5.0",
+    "appium-base-driver": "^2.18.4",
+    "appium-support": "^2.11.1",
     "asyncbox": "^2.0.2",
-    "babel-runtime": "=5.8.24",
-    "bluebird": "^3.1.1",
+    "babel-core": "5.8.24",
+    "babel-runtime": "5.8.24",
+    "bluebird": "^3.5.1",
     "lodash": "^3.9.1",
```

Comment:
can we also update lodash to 4+ like in other modules?

Reply:
There are number of places where we could update. https://github.com/appium/appium/issues/9866